### PR TITLE
New Port Post Void

### PR DESCRIPTION
**Pull request by Fraxinus88**
# New Port for Post Void

## Game Information
- **Title**: Post Void
- **URL**: https://store.steampowered.com/app/1285670/Post_Void/
### CFW Tests
Ensure your game has been tested on all major CFWs:
- [X] AmberELEC
- [X] ArkOS
- [X] JELOS

### Resolution Tests
Test all major resolutions:
- [X] 480x320
- [X] 640x480
- [X] 720x720 (RGB30)
- [X] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

**Comment by Fraxinus88** [comment]
Kindly check whether the cover.png is okay. Trying a cover :p

**Review by kloptops — CHANGES_REQUESTED** [review]
Cover.png requires a `gameinfo.xml` to be used.

I am working on a workflow/checker for these files at the moment so don't stress too much about them for now.

File: ports/postvoid/Post Void.sh [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs:$GAMEDIR/utils/libs"
+export GMLOADER_DEPTH_DISABLE=0
+export GMLOADER_SAVEDIR="$GAMEDIR/gamedata/"
+
+# We log the execution of the script into log.txt
+exec > >(tee "$GAMEDIR/log.txt") 2>&1
+
+cd $GAMEDIR
+
+if [ -f "${controlfolder}/libgl${CFWNAME}.txt" ]; then
```
[the remark this answers is not in the document]
Comment (Fraxinus88) [reply ports/postvoid/Post Void.sh]: Okies edited. Thanks for spotting this :)

**Review by kloptops — APPROVED** [review]
👍

**Pull request merged**